common/scala/src/main/scala/whisk/core/database/RemoteCacheInvalidation.scala

```diff
@@ -37,6 +37,7 @@ import whisk.core.entity.CacheKey
 import whisk.core.entity.InstanceId
 import whisk.core.entity.WhiskAction
 import whisk.core.entity.WhiskActionMetaData
+import whisk.core.WhiskConfig
```

Comment:
odd - this is now needed?

Reply:
removed. I thought it was still needed because WhiskConfig is mentioned in the constructor down below, but I guess I don't understand Scala's import rules...only have to import if you actually do something with the type (not just passing it through?)